src/api/auth.ts

```
@@ -0,0 +1,12 @@
+import { authApi } from './instance/authApi'
+
+export const reissue = async (refreshToken: string) => {
+  try {
+    const response = await authApi.post<string>('/auth/reissue', {
+      refreshToken,
+    })
+    return response.data
+  } catch (error) {
+    throw error
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)